entities/storobj/storage_object.go

```diff
@@ -145,6 +145,7 @@ func FromBinaryUUIDOnly(data []byte) (*Object, error) {
 	return ko, nil
 }
 
+// TODO multivector add multivector to addProp?
 func FromBinaryOptional(data []byte,
 	addProp additional.Properties, properties *PropertyExtraction,
```

Comment:
Should I add a new `MultiVectors` field to `addProp` or continue to use `Vectors` for both?

Reply:
Marcin says continue to use Vectors for both, we don't want to mess with `addProp` right now and a higher layer in the DB enforces that vector names are unique.